spec/features/two_factor_authenticatable_spec.rb

```diff
@@ -18,12 +18,12 @@
         visit new_user_session_path
         complete_sign_in_form_for(user)
 
-        expect(page).to have_content 'Enter your personal code'
+        expect(page).to have_content 'Enter the code that was sent to you'
 
         expect(SMSProvider.messages.size).to eq(1)
         message = SMSProvider.last_message
         expect(message.to).to eq(user.phone_number)
-        expect(message.body).to eq(user.otp_code)
+        expect(message.body.length).to eq(6)
```

Comment:
I will fix this.  The problem is that accessing the otp_code now requires a reload, since its stored in the database.   For some reason adding a reload here causes other tests to fail.